src/analysis/lattices/vector.h

```diff
@@ -39,13 +46,7 @@ template<Lattice L> struct Vector {
     return Element(size, lattice.getBottom());
   }
 
-  Element getTop() const noexcept
-#if __cplusplus >= 202002L
-    requires FullLattice<L>
-#endif
```

Comment:
What happened here?

Reply:
This wasn't pulling its weight, complexity-wise, so I removed it as a drive-by.